Something on musllinux is [off](https://github.com/numpy/numpy/actions/runs/7723072436/job/21052421270?pr=25732#step:4:840), the string replacement function `get_library()` is not being called when generating the pkg-config file. Is this from the wheel itself or from the build step...
```
ld: cannot find -l{get_library()}
```

--- a/.spin/cmds.py
+++ b/.spin/cmds.py
@@ -565,7 +565,9 @@ def _config_openblas(blas_variant):
             fid.write(f"import {module_name}\n")
         os.makedirs(openblas_dir, exist_ok=True)
         with open(pkg_config_fname, "wt", encoding="utf8") as fid:
-            fid.write(openblas.get_pkg_config().replace("\\", "/"))
+            fid.write(
+                openblas.get_pkg_config(use_preloading=True).replace("\\", "/")
+            )
 
 
 @click.command()

--- a/requirements/ci32_requirements.txt
+++ b/requirements/ci32_requirements.txt
@@ -1,4 +1,4 @@
 spin
 # Keep this in sync with ci_requirements.txt
-scipy-openblas32<=0.3.26.0
+scipy-openblas32<=0.3.26.0.1
 

--- a/requirements/ci32_requirements.txt
+++ b/requirements/ci32_requirements.txt
@@ -1,4 +1,4 @@
 spin
 # Keep this in sync with ci_requirements.txt
-scipy-openblas32<=0.3.26.0.1
+scipy-openblas32==0.3.26.0.2
 

--- a/requirements/ci_requirements.txt
+++ b/requirements/ci_requirements.txt
@@ -1,4 +1,4 @@
 spin
 # Keep this in sync with ci32_requirements.txt
-scipy-openblas32<=0.3.26.0
-scipy-openblas64<=0.3.26.0
+scipy-openblas32<=0.3.26.0.1
+scipy-openblas64<=0.3.26.0.1

--- a/requirements/ci_requirements.txt
+++ b/requirements/ci_requirements.txt
@@ -1,4 +1,4 @@
 spin
 # Keep this in sync with ci32_requirements.txt
-scipy-openblas32<=0.3.26.0.1
-scipy-openblas64<=0.3.26.0.1
+scipy-openblas32==0.3.26.0.2
+scipy-openblas64==0.3.26.0.2

